framework/src/org/apache/cordova/CoreAndroid.java

```diff
@@ -357,4 +358,33 @@ public void sendResumeEvent(PluginResult resumeEvent) {
             }
         }
     }
+
+      /*
+     * This needs to be implemented if you wish to use the Camera Plugin or other plugins
+     * that read the Build Configuration.
+     *
+     * Thanks to Phil@Medtronic and Graham Borland for finding the answer and posting it to
+     * StackOverflow.  This is annoying as hell!
+     *
+     */
+
+    public static Object getBuildConfigValue(Context ctx, String key)
```

Comment:
Notice:  if you want to use this method within a ProGuard-obfuscated build the application will exit with `java.lang.ClassNotFoundException`